code/modules/mob/living/living.dm

```diff
@@ -719,13 +718,18 @@
 	. = TRUE
 	if(pulledby.grab_state || resting)
 		var/altered_grab_state = pulledby.grab_state
-		if(resting) //If resting, resisting out of a grab is equivalent to 1 grab state higher
+		if(resting && pulledby.grab_state < 3) //If resting, resisting out of a grab is equivalent to 1 grab state higher. wont make the grab state exceed the normal max, however
```

Comment:
grab states should have defines 100%

Reply:
Turns out defines for this already existed, used em